curriculum/challenges/english/25-front-end-development/workshop-fruit-search-app/67f8ef57b9974052e5ef6b30.md

```diff
@@ -0,0 +1,120 @@
+---
+id: 67f8ef57b9974052e5ef6b30
+title: Step 10
+challengeType: 0
+dashedName: step-10
+---
+
+# --description--
+
+Use the ternary operator inside the `results` `div` to conditionally display content based on the `results` array. When the array contains items, display each fruit name inside a `div` with a class of `result-item`. When the array is empty, display a message that reads exactly: `No results found`.
+
+# --hints--
+
+You should use the ternary operator to conditionally render the results when `results.length > 0`, and render a `<p>` element with the text `No results found` when the array is empty.
```

Comment:
this should be broken up into separate tests. one to check for the results and another to check for when the array is empty and the message is shown. Also, we should find another way to test this instead of regex. reach out in chat for help and ideas for this.